.github/workflows/ci.yml

```diff
@@ -35,15 +35,15 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu, windows]
-        node-version: [12.x, 14.x]
+        node-version: [16, 18, 20]
 
     steps:
     - uses: actions/checkout@v3
     - uses: actions/setup-node@v3
       with:
         node-version: ${{ matrix.node-version }}
     - name: install dependencies
-      run: yarn install --ignore-engines --frozen-lockfile
+      run: yarn install --ignore-engines --frozen-lockfile --network-timeout 600000
```

Comment:
Yarn installs on windows often fail, which is pretty time consuming.
This should hopefully improve that.